src/command.ts

```diff
@@ -59,49 +71,48 @@ export type ComparisonResult = {
   percentage: number
 }
 
-export type UpdateBaseResult = {
-  baseUpdated: boolean
-}
-
 /** Call the plugin to compare snapshot images and generate a diff */
 function compareScreenshots(name: string, screenshotOptions: any): Chainable<ComparisonResult> {
-  const errorThreshold = getErrorThreshold(screenshotOptions)
-  const options: CompareSnapshotsPluginArgs = {
-    fileName: name,
-    errorThreshold,
-    // @ts-expect-error TODO fix potential null error
-    specRelativePath: Cypress.config().spec.relative,
-    specFolder: Cypress.env('visualRegression').specFolder,
-    baseDirectory: Cypress.env('visualRegression').baseDirectory,
-    diffDirectory: Cypress.env('visualRegression').diffDirectory,
-    generateDiff: Cypress.env('visualRegression').generateDiff,
-    failSilently: false
-  }
-
-  if (screenshotOptions.failSilently !== null) {
-    options.failSilently = screenshotOptions.failSilently
-  } else if (Cypress.env('failSilently') !== null) {
-    options.failSilently = Cypress.env('failSilently')
-  }
+  return cy.get('@screenshotAbsolutePath').then((screenshotAbsolutePath: unknown) => {
+    if (typeof screenshotAbsolutePath !== 'string') {
+      throw new Error('Could not resolve screenshot path')
+    }
+    const errorThreshold = getErrorThreshold(screenshotOptions)
+    const options: CompareSnapshotsPluginArgs = {
+      screenshotName: name,
+      errorThreshold,
+      // @ts-expect-error TODO fix potential null error
+      specName: Cypress.config().spec.name,
+      screenshotAbsolutePath,
```

Comment:
so that here we have a bit cleaner values, and names are more intuitive.
For getting the screenshot we only need the absolute path (this conveniently solves the problem where the screenshot path is different in run vs open mode in cypress)

And to store the base screenshot, we need the name and base directory as mandatory, and also spec file name to organize it better